libpod/define/exec_codes.go

```diff
@@ -46,6 +46,7 @@ func ExitCode(err error) int {
 	e := strings.ToLower(err.Error())
 	logrus.Debugf("ExitCode msg: %q", e)
 	if strings.Contains(e, "not found") ||
+		strings.Contains(e, "executable path is empty") ||
```

Comment:
it might make sense to link the specific crun PR here

Reply:
that can be retrieved from `git blame`, if anyone cares